src/plugins/usage_collection/server/routes/stats/README.md

```diff
@@ -10,7 +10,7 @@ However, the information detailed above can be extended, with the combination of
 |:----------------|:-------------:|:------------|
 |`extended`|`false`|When `true`, it adds `clusterUuid` and `usage`. The latter contains the information reported by all the Usage Collectors registered in the Kibana server. It may throw `503 Stats not ready` if any of the collectors is not fully initialized yet.|
 |`legacy`|`false`|By default, when `extended=true`, the key names of the data in `usage` are transformed into API-friendlier `snake_case` format (i.e.: `clusterUuid` is transformed to `cluster_uuid`). When this parameter is `true`, the data is returned as-is.|
-|`exclude_usage`|`false`|When `true`, and `extended=true`, it will report `clusterUuid` but no `usage`.|
+|`exclude_usage`|`true`|When `true`, and `extended=true`, it will report `clusterUuid` but no `usage`.|
```

Comment:
```suggestion
|`exclude_usage`|`true`|When `true`, and `extended=true`, it will report `clusterUuid` but no `usage`. (Note: even if set to `false`, Kibana dismisses the parameter and acts as if it's always true) | 
```

Reply:
Good call! Thank you for pointing it out!